runelite-client/src/test/java/net/runelite/client/plugins/cooking/CookingPluginTest.java

```diff
@@ -60,7 +60,8 @@
 		"You roast a lobster.",
 		"You cook a bass.",
 		"You successfully bake a tasty garden pie.",
-		"You dry a piece of meat and extract the sinew."
+		"You dry a piece of meat and extract the sinew.",
+		"You burn some marrentill in the incense burner."
```

Comment:
Don't add this to the successful cooking messages. Define and test it separately from this array.

Reply:
Changed as requested.